.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -8,15 +8,15 @@
         <value>
           <list size="5">
             <item index="0" class="java.lang.String" itemvalue="2.7" />
-            <item index="1" class="java.lang.String" itemvalue="3.3" />
-            <item index="2" class="java.lang.String" itemvalue="3.4" />
-            <item index="3" class="java.lang.String" itemvalue="3.5" />
-            <item index="4" class="java.lang.String" itemvalue="3.6" />
+            <item index="1" class="java.lang.String" itemvalue="3.4" />
```

Comment:
I'm not a fan of versioning this kind of directory, but it's up to you :)

Reply:
It's historic not removed part. Probably later will be removed, while now just updated to the correct supported platforms